packages/x-components/src/x-modules/identifier-results/store/__tests__/actions.spec.ts

```diff
@@ -1,47 +1,65 @@
 import { createLocalVue } from '@vue/test-utils';
 import Vuex, { Store } from 'vuex';
-import { map } from '../../../../utils';
 import { getResultsStub } from '../../../../__stubs__/results-stubs.factory';
 import { getMockedAdapter, installNewXPlugin } from '../../../../__tests__/utils';
+import { SafeStore } from '../../../../store/__tests__/utils';
 import { identifierResultsXStoreModule } from '../module';
-import { IdentifierResultsState } from '../types';
+import {
+  IdentifierResultsActions,
+  IdentifierResultsGetters,
+  IdentifierResultsMutations,
+  IdentifierResultsState
+} from '../types';
 import { resetIdentifierResultsStateWith } from './utils';
 
 describe('testing identifier results module actions', () => {
   const mockedResults = getResultsStub();
   const adapter = getMockedAdapter({ searchById: { results: mockedResults } });
 
-  const actionKeys = map(identifierResultsXStoreModule.actions, action => action);
```

Comment:
I like the use of this map in order to know which actions the module has and test all of them, why did you remove them?

Reply:
no necessary any more thanks to TS